<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KstChangeFileDialog</class>
 <widget class="QDialog" name="KstChangeFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Data File</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>11</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="1" column="0">
    <widget class="QGroupBox" name="buttonGroup2">
     <property name="title">
      <string>New Data File</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="textLabel1">
          <property name="text">
           <string>&amp;File:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>_dataFile</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout">
          <item>
           <widget class="KUrlRequester" name="_dataFile" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="_fileType">
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer25">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>71</width>
            <height>31</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="_configureSource">
          <property name="text">
           <string>Con&amp;figure...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QRadioButton" name="_changeSelected">
          <property name="text">
           <string>C&amp;hange selected vectors and matrices</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="spacer3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>16</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout">
        <item row="0" column="2">
         <spacer name="spacer4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>16</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="_duplicateDependents">
          <property name="text">
           <string>D&amp;uplicate dependents</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="_duplicateSelected">
          <property name="text">
           <string>&amp;Duplicate selected vectors and matrices</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout">
     <item>
      <spacer name="Spacer15">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>180</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ChangeFileApply">
       <property name="whatsThis">
        <string>Apply new source file to selected vectors.</string>
       </property>
       <property name="text">
        <string>&amp;Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ChangeFileOK">
       <property name="whatsThis">
        <string>Apply new source file to selected vectors.</string>
       </property>
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ChangeFileCancel">
       <property name="whatsThis">
        <string>Close dialog without applying further changes.</string>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="buttonGroup1">
     <property name="title">
      <string>Vector and Matrix Selection</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QPushButton" name="_allFromFile">
          <property name="text">
           <string>All &amp;From</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="_files">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QGridLayout">
        <item row="0" column="0" rowspan="5">
         <widget class="QListWidget" name="ChangeFileCurveList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="whatsThis">
           <string>The source file for the vectors selected in this list will be changed.</string>
          </property>
          <property name="SelectionMode">
           <enum>QListWidget::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="ChangeFileClear">
          <property name="whatsThis">
           <string>Unselect all vectors in the list.</string>
          </property>
          <property name="text">
           <string>C&amp;lear</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <spacer name="spacer2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>80</width>
            <height>51</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="ChangeFileSelectAll">
          <property name="whatsThis">
           <string>Select all vectors in the list.</string>
          </property>
          <property name="text">
           <string>S&amp;elect All</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="_clearFilter">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="_filter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_allFromFile</tabstop>
  <tabstop>_files</tabstop>
  <tabstop>_clearFilter</tabstop>
  <tabstop>_filter</tabstop>
  <tabstop>ChangeFileCurveList</tabstop>
  <tabstop>ChangeFileClear</tabstop>
  <tabstop>ChangeFileSelectAll</tabstop>
  <tabstop>_dataFile</tabstop>
  <tabstop>_changeSelected</tabstop>
  <tabstop>_duplicateSelected</tabstop>
  <tabstop>_duplicateDependents</tabstop>
  <tabstop>ChangeFileApply</tabstop>
  <tabstop>ChangeFileCancel</tabstop>
 </tabstops>
 <includes>
  <include location="local">kurlrequester.h</include>
  <include location="local">klineedit.h</include>
  <include location="local">kpushbutton.h</include>
  <include location="local">qcombobox.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>ChangeFileCancel</sender>
   <signal>clicked()</signal>
   <receiver>KstChangeFileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
